src/Gwen/APIHandlers/SpotifyAPI/cred.py

```diff
@@ -0,0 +1,5 @@
+client_id='4c7713b205ba4ff097e50ff00315bc1c'
+client_secret='2e9a3964684f48f895f60ce0ccd7bed7'   
+redirect_uri='http://google.com/'
+user=''
+passw=''
```

Comment:
Convert this python file to JSON and read in variables using src/utils.py -> convert_path_to_JSON (going to change the name).

Reply:
I did it but I don't know how to import utils.py in my program. I tried "from utils import convert_path_to_JSON" but it says no modules found

Reply:
import src.utils